src/js/calculator/calculateOperation.js

```diff
@@ -0,0 +1,23 @@
+import { OPERATION } from "./constant.js";
+import { isValidOperation } from "./validateOperation.js";
+
+function calculateOperation() {
+  const usedOperator = isValidOperation();
+  const currentOperation = OPERATION.innerText;
+  const operations = currentOperation.split(usedOperator);
+
+  const operand1 = Number(operations[0]);
+  const operand2 = Number(operations[1]);
+
+  if (usedOperator === "+") {
+    OPERATION.innerText = operand1 + operand2;
+  } else if (usedOperator === "-") {
+    OPERATION.innerText = operand1 - operand2;
+  } else if (usedOperator === "X") {
+    OPERATION.innerText = operand1 * operand2;
+  } else {
+    OPERATION.innerText = operand1 / operand2;
```

Comment:
```suggestion
  const operationExecutor = {
   "+": (operand1, operand2) => operand1 + operand2,
   "-": (operand1, operand2) => operand1 - operand2
   "X": (operand1, operand2) => operand1 * operand2
   "/": (operand1, operand2) => operand1 / operand2 
  }
  OPERATION.innerText = operationExecutor[usedOperator]
```

if else 문을 통해 분기도 좋지만 이렇게 테이블을 만들어 단순하게 분기하는 방법도 많이 사용하니 참고해주세요~

Reply:
감사합니다. 이렇게 작성하면 indent depth 도 줄어들고 가독성도 훨씬 좋네요.
앞으로 코드 작성시 분기가 생길 경우 꼭 알려주신 방법으로 구현해보겠습니다.